src/utils/notifications.ts

```diff
@@ -150,7 +147,9 @@ export function getMarkedUnreadState(room: Room): boolean | undefined {
 }
 
 /**
- * Sets the marked_unread state of the given room
+ * Sets the marked_unread state of the given room. This sets some room account data that indicates to
+ * clients that the user considers this room to be 'unread', but without any actual notifications.
```

Comment:
> but without any actual notifications.

I don't understand this bit. Why would setting the room account data cause notifications?

Reply:
It doesn't, but clients implementing the MSC will see it and draw a notification badge on the room even though, as far as push rule's calculation of notifications a a function of messages in the room + read receipts goes, there are no notifications.